auth-server/src/controllers/user.ts

```diff
@@ -58,7 +58,8 @@ export const postSignup = async (req, res) => {
 			refreshToken,
 			{
 				httpOnly: true,
-				maxAge: 6 * 30 * 24 * 60 * 60 * 1000 // 6 months
+				maxAge: 6 * 30 * 24 * 60 * 60 * 1000, // 6 months
+				samesite: 'strict'
```

Comment:
need to be done above in login also. Lets move setting this cookie in seperate function so we do not have to do 2 times